qutebrowser/config/configdata.yml

```diff
@@ -2029,7 +2034,7 @@ fonts.web.size.minimum_logical:
     name: Int
     minval: 0
     maxval: maxint
-  desc: The minimum logical font size that is applied when zooming out.
+  desc: Minimum logical font size (in points) that is applied when zooming out.
```

Comment:
Looking at [the Qt docs](http://doc.qt.io/qt-5/qwebenginesettings.html#setFontSize), I think those are all in pixels?

Reply:
good catch, I remeber looking it uo but must have been asleep.